src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1373,8 +1384,8 @@ public function __call($method, $parameters)
             return call_user_func_array(static::$macros[$method], $parameters);
         }
 
-        if (method_exists($this->model, $scope = 'scope'.ucfirst($method))) {
-            return $this->callScope([$this->model, $scope], $parameters);
+        if ($this->hasScope($method)) {
```

Comment:
Kinda odd that hasScope expects the scope name and not the method name, but callScope expects the actual method name. (don't change anything yet please. I am writing this comment to be discussed)

Reply:
I did find that weird, but wanted to change the minimum amount possible, and since changing how `callScope` works would be considered a breaking change, wanted to delegate that to a future PR (if desired).